eth/fetcher/block_fetcher.go

```diff
@@ -795,6 +806,7 @@ func (f *BlockFetcher) importBlocks(peer string, block *types.Block) {
 		parent := f.getBlock(block.ParentHash())
 		if parent == nil {
 			log.Debug("Unknown parent of propagated block", "peer", peer, "number", block.Number(), "hash", hash, "parent", block.ParentHash())
```

Comment:
You should sleep for a while before enquene again.

Reply:
added